src/seer/automation/agent/client.py

```diff
@@ -488,6 +488,7 @@ def is_completion_exception_retryable(exception: Exception) -> bool:
             "overloaded_error",
             "Internal server error",
             "not_found_error",
+            "404",
```

Comment:
"404" seems a bit too liberal

```suggestion
            "404, 'message': 'Publisher Model",
```

Reply:
also just curious why we think this is flaky or if we're just retrying to see if it is